src/main/scala/io/appthreat/atom/Atom.scala

```diff
@@ -726,22 +726,26 @@ object Atom:
   end handleScalaSemantics
 
   private def createJsSrc2Cpg(config: AtomConfig, outputAtomFile: String): Try[Cpg] =
-      new JsSrc2Cpg()
-          .createCpgWithOverlays(
-            JSConfig()
-                .withDisableDummyTypes(true)
-                .withTypePropagationIterations(TYPE_PROPAGATION_ITERATIONS)
-                .withInputPath(config.inputPath.pathAsString)
-                .withOutputPath(outputAtomFile)
-          )
-          .map { ag =>
-            new JavaScriptInheritanceNamePass(ag).createAndApply()
-            new ConstClosurePass(ag).createAndApply()
-            new ImportResolverPass(ag).createAndApply()
-            new JavaScriptTypeRecoveryPass(ag).createAndApply()
-            new TypeHintPass(ag).createAndApply()
-            ag
-          }
+    val initialConfig = JSConfig()
+        .withDisableDummyTypes(true)
+        .withTypePropagationIterations(TYPE_PROPAGATION_ITERATIONS)
+        .withInputPath(config.inputPath.pathAsString)
+        .withOutputPath(outputAtomFile)
+        .withFlow(config.language.equalsIgnoreCase("FLOW"))
+    val finalConfig = sys.env.get("CHEN_ASTGEN_OUT") match
```

Comment:
Support for new environment variable `CHEN_ASTGEN_OUT` to reuse an existing astgen output directory.